libs/judicial-system/types/src/lib/case.ts

```diff
@@ -273,6 +273,16 @@ export const getStatementDeadline = (appealReceived: Date): string => {
   ).toISOString()
 }
 
+export const isAppealableDecision = (decision?: CaseAppealDecision | null) => {
```

Comment:
This implementation is fine as it is, but it would also be possible to add properties to the case in the api transformer. A property called `canBeAppealed` is already added there and instead of exposing this function, two new properties could be added `prosecutorCanAppeal` and `defenderCanAppeal`.

Both implementations are valid, but I feel adding new properties to the case would be more consistent with the existing implemention.

Reply:
True, I like that idea, I'm going to change to doing that for consistency